reframe/core/fields.py

```diff
@@ -110,8 +110,7 @@ class TimerField(TypedField):
     '''Stores a timer in the form of a :class:`datetime.timedelta` object'''
 
     def __init__(self, fieldname, *other_types):
-        super().__init__(fieldname, datetime.timedelta, str,
-                         types.Tuple[int, int, int], *other_types)
+        super().__init__(fieldname, datetime.timedelta, str, *other_types)
 
     def __set__(self, obj, value):
         self._check_type(value)
```

Comment:
Shouldn't the check that is just after this line be deleted?
```
    if isinstance(value, tuple):
            user_deprecation_warning(
            user_deprecation_warning(
                "setting a timer field from a tuple is deprecated: "
                "setting a timer field from a tuple is deprecated: "
                "please use a string '<days>d<hours>h<minutes>m<seconds>s'")
                "please use a string '<days>d<hours>h<minutes>m<seconds>s'")
            h, m, s = value
            h, m, s = value
            value = datetime.timedelta(hours=h, minutes=m, seconds=s)
            value = datetime.timedelta(hours=h, minutes=m, seconds=s)
```

Reply:
Oh, yes! I don't know why I forgot that. Perhaps I forgot to push.